project-base/storefront/components/Pages/PersonalData/Overview/PersonalDataOverviewContent.tsx

```
@@ -52,13 +49,13 @@ export const PersonalDataOverviewContent: FC<PersonalDataOverviewContentProps> =
-                <h1>{t('Personal data overview')}</h1>
-
-                {contentSiteText && (
-                    <div className="[&_section]:text-justify" id="personal-data-overview-content">
-                        <UserText htmlContent={contentSiteText} />
-                    </div>
-                )}
+                <PageHero
+                    icon={UserIcon}
+                    title={t('Personal data overview')}
+                    description={
+                        contentSiteText ? <span dangerouslySetInnerHTML={{ __html: contentSiteText }} /> : undefined
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)